### PR TITLE
Adding recipe for TPC FEC utils v0.1.0

### PR DESCRIPTION
**Pull request by sk3x**

### CODE REVIEW

**Comment by awegrzyn** [comment]
I'm not sure that storing GitLab token as clear text is good idea. @ktf Would it be possible to add it as GitHub Secret?

**Comment by awegrzyn** [comment]
@vascobarroso As this will be running at FLP shall we add it to `@alisw/dataflow-experts` in CODEWONERS?

File: tpcfecutils.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+cp ${BUILDDIR}/compile_commands.json ${INSTALLROOT}
+
+# Reduce number of jobs if invoked by Jenkins
+if [ ! "X$JENKINS_HOME" = X ]; then
+  JOBS=1
+fi
+make ${JOBS+-j $JOBS} install
+
+#ModuleFile
+mkdir -p etc/modulefiles
```
Comment (ktf) [inline tpcfecutils.sh:51]: I would suggest you use the new `alibuild-generate-module` utility which you can find in `alibuild-recipe-tools`. Have a look at e.g. the `boost.sh` recipe to see how it's used.

Comment (ktf) [reply tpcfecutils.sh:51]: I will merge this, you can fix it later.

**Comment by ktf** [comment]
Merging since nothing depends on it yet.

**Pull request merged**